src/main/java/com/puppycrawl/tools/checkstyle/checks/coding/SuperFinalizeCheck.java

```diff
@@ -39,6 +39,21 @@
  * <pre>
  * &lt;module name=&quot;SuperFinalize&quot;/&gt;
  * </pre>
+ * <p>Example:</p>
+ * <pre>
+ * public class Test {
+ *     protected void finalize() throws Throwable {
+ *        try {
+ *            System.out.println("Finalize of Sub Class"); // Release resources, perform cleanup
+ *        } catch(Throwable t){
+ *            throw t;
+ *        } finally{
+ *            System.out.println("Calling finalize of Super Class");
+ *            super.finalize();
+ *        }
+ *     }
+ * }
+ * </pre>
```

Comment:
Please add one more example with violation. To let people see not good example

Reply:
Yeah sure

Reply:
![image](https://user-images.githubusercontent.com/70961605/166206332-65154f11-9952-4a10-be54-a631c3370f3c.png)
![image](https://user-images.githubusercontent.com/70961605/166206380-0ffe2746-5600-4ca8-a226-a719552a0741.png)

The number of characters and spaces used are exact in both config_coding.xml and SuperFinalizeCheck.java file but I am unable to resolve the error 
- Parent is {@code com.puppycrawl.tools.checkstyle.TreeWalker}
        at com.puppycrawl.tools.checkstyle.internal.XdocsJavaDocsTest.examineCheckSection(XdocsJavaDocsTest.java:191)
        at com.puppycrawl.tools.checkstyle.internal.XdocsJavaDocsTest.testAllCheckSectionJavaDocs(XdocsJavaDocsTest.java:164)

Also I'm unable to find as the above displayed path is not found on my machine "XdocsJavaDocsTest.java"
Please help

Reply:
First image shows that sources are different, looks attentively to spot a diff